packages/specs/source/protocols/uninstall-virtual-app.md

```diff
@@ -8,13 +8,12 @@ Three users run the protocol. They are designated as `initiator`, `responder`, a
 
 ## The `UninstallVirtualAppParams` type
 
-
-|            Field             |   type    |                         description                         |
-| ---------------------------- | --------- | ----------------------------------------------------------- |
+| Field                       | type      | description                                                |
```

Comment:
Please format this table with the visual studio extension

Reply:
It is formatted